bedita-app/app_controller.php

```diff
@@ -1232,9 +1232,6 @@ protected function paginatedList($id, $filter, $order, $dir, $page, $dim) {
             }
         }
 
-        // get Tags
-        $listTags = ClassRegistry::init("Category")->getTags(array("cloud" => false));
-
         // template data
```

Comment:
I'd remove or deprecate `Category::getTags`

Reply:
It is used elsewhere.